src/lib/components/events/event-item.svelte

```diff
@@ -13,6 +13,15 @@
   let anchor: HTMLElement;
   let details: HTMLDetailsElement;
 
+  // Slicing the date param to extract the end time
+  function addingEndTime(endTime) {
+    endTime = endTime.slice(endTime.length - 25, endTime.length - 21);
+    endTime += ' PM';
+    return endTime;
+  }
+
+  const endTime = addingEndTime(info.date);
+
```

Comment:
If the `endTime` is added to the `AcmEvent` interface, that would make things simpler!

To accomplish this, you could add this change to our API endpoint below to your PR.

```diff
+++ b/src/lib/ical/utils.ts
@@ -7,7 +7,8 @@ import { parseBool } from '$lib/common/utils';
 export interface AcmEvent {
   month: string;
   day: number;
-  time: string;
+  startTime: string;
+  endTime: string;
   hasStarted: boolean;
   hasEnded: boolean;
   isPinned: boolean;
@@ -304,7 +305,8 @@ export function makeAcmEvent(
   const date = dtStart.toString();
   const month = dtStart.toLocaleString('en-US', { month: 'long' });
   const day = dtStart.day;
-  const time = dtStart.toLocaleString('en-US', { hour: 'numeric', minute: 'numeric' });
+  const startTime = dtStart.toLocaleString('en-US', { hour: 'numeric', minute: 'numeric' });
+  const endTime = dtEnd.toLocaleString('en-US', { hour: 'numeric', minute: 'numeric' });
   const hasStarted = Temporal.ZonedDateTime.compare(referenceDate, dtStart) >= 0;
   const hasEnded = Temporal.ZonedDateTime.compare(referenceDate, dtEnd) >= 0;
   const duration = dtEnd.since(dtStart).minutes + ' minutes';
@@ -353,7 +355,8 @@ export function makeAcmEvent(
   return {
     month,
     day,
-    time,
+    startTime,
+    endTime,
     date,
     hasStarted,
     hasEnded,
```

This way, the start and end times could be accessed in the HTML via `info.startTime` and `info.endTime` respectively.